src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -231,6 +233,34 @@ def sync_project(self) -> None:
             stderr=subprocess.PIPE,
         )
 
+        server_status: Result = self.client.get_server_status()
+        if server_status.code != 200:
+            raise Exception(f"Failed to get server status for {self._host}")
+        server_version: Version
+        try:
+            server_version = Version(server_status.result["data"]["version"])
+        except (KeyError, TypeError):
+            raise Exception("Unexpected response for server status API call")
+        # iso5 requires explicit project installation
+        if server_version >= Version("5.dev"):
+            venv_path: str = os.path.join(server_path, ".env")
+            # venv might not exist yet so can't just access its `inmanta` executable -> install via Python script instead
```

Comment:
If the venv is not created, will this just work? Isn't it necessary to call `.use_virtual_env` on the `VirtualEnv` object created by the project?  I am genuinely asking, I failed to verify this.  If you know it is the case, you can simply close this, I do not think that a test case is necessary for this.

Reply:
I am satisfied indeed! :p